.gitignore

```diff
@@ -1 +1,2 @@
-/vendor
\ No newline at end of file
+/vendor
+.idea
```

Comment:
Could you move this to your global `.gitignore` instead of the repos?

Reply:
You're right - moved to global .gitignore. Thanks!